attendees_and_learners.rst

```diff
@@ -9,7 +9,7 @@ Workshops
 =========
 DjangoGirls in Portland, 27th August 2015
 ------------------------------------------
-
+Natalie Hansen
```

Comment:
Hi Natalie. 

This needs to be in the format:

    * Natalie Hansen

Can you amend your file, and go through the add/commit/cycle again (just push to the same branch) so I can accept it?